src/fontManager.h

```diff
@@ -45,11 +45,18 @@ class FontManager {
 	nameColorsMap fonts;
 	
 public:
+	FontManager();
 	FontManager(Graphics* graphics);
+	FontManager(const FontManager& fm);	// copy constructor
+	//FontManager& operator=(const FontManager& rhs) {};
```

Comment:
Going to remove to comment. The comment allows for this
`FontManager newFm = oldFm;`